Cargo.toml

```diff
@@ -24,7 +24,7 @@ integer128 = []
 
 [dependencies]
 base64 = "0.13"
-bitflags = "=1.2.1"
+bitflags = "1.0.4"
```

Comment:
What’s special about V1.0.4?

Reply:
oh that was just the version we had required initially (I just did git revert)